### ISSUE TITLE
`package.json` is referencing non existing file

### ISSUE THREAD

**Issue by zhuhaow** [qwikest/icons#1]
First of all, thank you. This package is SUPER helpful.

Currently, the `package.json` is referencing wrong files
https://github.com/qwikest/icons/blob/ad582938a25aa8d84fdb0517d2a56dbad8fff0e9/package.json#L14-L16

causing error

```
thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: Opening /......./node_modules/.pnpm/@qwikest+icons@0.0.1_@builder.io+qwik@0.19.2/node_modules/@qwikest/icons/lib/index.qwik.mjs

Caused by:
    No such file or directory (os error 2)', packages/qwik/src/napi/src/lib.rs:21:50
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
✓ 0 modules transformed.
[vite-plugin-qwik] called `Result::unwrap()` on an `Err` value: Opening /......./node_modules/.pnpm/@qwikest+icons@0.0.1_@builder.io+qwik@0.19.2/node_modules/@qwikest/icons/lib/index.qwik.mjs

Caused by:
    No such file or directory (os error 2)
error during build:
Error: called `Result::unwrap()` on an `Err` value: Opening /......./node_modules/.pnpm/@qwikest+icons@0.0.1_@builder.io+qwik@0.19.2/node_modules/@qwikest/icons/lib/index.qwik.mjs

```

**Comment by NiklasPor** [comment]
Thanks, totally forgot about those when I created the named exports. The named exports are working fine for you, right?

**Comment by NiklasPor** [comment]
This should be resolved in `0.0.2`. Also added [Iconoir](https://iconoir.com/) and [Mono Icons](https://icons.mono.company/) 👋 

If you reencounter this issue, feel free to reopen 👍

**Comment by zhuhaow** [comment]
Sorry for the late reply.

Confirm it works fine now. Thanks!
